source/CardGames.py

```diff
@@ -324,28 +335,81 @@ def display_board_general(self, cards_in_play):
 
 
     def fugitive_first_turn(self):
-      fugitive_deck = self.starting_cards
-      fugitive_deck.append(self.escape_card)
+      self.fugitive_deck = self.starting_cards
```

Comment:
We will probably need to use the player object's deck instead of a global variable in the game_setup script.